x-pack/plugins/enterprise_search/public/applications/app_search/components/settings/settings.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { EuiPageHeader, EuiPageHeaderSection, EuiPageContentBody, EuiTitle } from '@elastic/eui';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { FlashMessages } from '../../../shared/flash_messages';
+
+export const Settings: React.FC = () => {
+  return (
+    <>
+      <SetPageChrome
+        trail={[
+          i18n.translate('xpack.enterpriseSearch.appSearch.settings.title', {
+            defaultMessage: 'Settings',
+          }),
+        ]}
+      />
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.settings.title', {
+                defaultMessage: 'Settings',
```

Comment:
Not a blocker, note to self to follow up in a future tech debt item for multiple views - we tend to repeat page titles in 3 different places (nav link, breadcrumbs, & page heading) - I should figure out a sane way to DRY that out some day for all our pages